src/main/java/se/sundsvall/casedata/integration/db/model/Note.java

```diff
@@ -55,6 +58,9 @@ public class Note extends BaseEntity {
 	@Column(name = "municipality_id")
 	private String municipalityId;
```

Comment:
Do we really need namespace here? Note seems to be a subtable to Errand (where namespace is defined)?

(This comment is probably applicable for other entities in this PR as well?)

Reply:
There exists resources to `GET`, `PATCH`, `PUT` and `DELETE` note without any connection to the Errand. So my reasoning is that because of that they need namespace (and municipalityId) 

or 

refactor the API and move the `GET`, `PATCH`, `PUT` and `DELETE` methods to 
`/{municipalityId}/{namespace}/errands/{errandId}/notes/{noteId}` 
instead of
`/{municipalityId}/{namespace}/notes/{noteId}`

Reply:
Yes, refactor the API according to your first suggestion would be the best, i.m.h.o., since it would resemble SupportManagament:s API.

Reply:
Okay. I'll refactor the API